src/frameworks/django/index.ts

```diff
@@ -0,0 +1,86 @@
+import { copy, mkdirp, pathExists } from "fs-extra";
+import { mkdir, readFile, readdir, writeFile } from "fs/promises";
+import { join } from "path";
+import { BuildResult, FrameworkType, SupportLevel } from "../interfaces";
+import { dirExistsSync } from "../../fsutils";
+import { findPythonCLI, hasPipDependency, spawnPython } from "../utils";
+import { spawnSync } from "child_process";
+
+export const name = "Django";
+export const support = SupportLevel.Experimental;
+export const type = FrameworkType.Framework;
+
+export async function discover(cwd: string) {
+  if (!hasPipDependency("Django", { cwd })) return;
+  if (!(await pathExists(join(cwd, "manage.py")))) return;
+  return { mayWantBackend: true, publicDirectory: "static" };
+}
+
+export function build(): Promise<BuildResult> {
+  return Promise.resolve({ wantsBackend: true });
+}
+
+export async function init(setup: any, config: any) {
+  const cwd = join(config.projectDir, setup.hosting.source);
+  await mkdirp(cwd);
+  const cli = findPythonCLI();
+  spawnSync(cli, ["-m", "venv", "venv"], { stdio: "ignore", cwd });
+  writeFile(join(cwd, "requirements.txt"), "Django");
+  await spawnPython("pip", ["install", "-r", "requirements.txt"], cwd);
+  await spawnPython("django-admin", ["startproject", setup.projectId.replace("-", "_"), "."], cwd);
```

Comment:
this should be `replaceAll`, explains @leoortizz  init issue—ok closing my laptop now, lolol, just had to check on a couple things :P